### ISSUE TITLE
PCIe test spits thousands of "Error-Input address is not aligned. Masking the last 2 bits"

### ISSUE THREAD

**Issue by hrw** [ARM-software/sbsa-acs#186]
SBSA-ACS v3.1 fails to work on "sbsa-ref" QEMU machine:

```
      *** Starting PCIe tests ***
 401 : Check ECAM Presence               : Result:  PASS
 402 : Check ECAM value in MCFG table    : Result:  PASS
 403 : Check ECAM Memory accessibility
  Error-Input address is not aligned. Masking the last 2 bits

  Error-Input address is not aligned. Masking the last 2 bits
```
That 'Error-Input' message is output hundreds of times. I just rebooted machine.

SBSA v3.0 works:

```
      *** Starting PCIe tests ***
 401 : Check ECAM Presence               : Result:  PASS
 402 : Check ECAM value in MCFG table    : Result:  PASS
 403 : Check ECAM Memory accessibility   : Result:  PASS
 420 : Check Type 0/1 common config rules: Result:  PASS
 421 : Check Type 0 config header rules  : Result:  PASS

```

**Comment by hrw** [comment]
And it passed anyway:
![obraz](https://user-images.githubusercontent.com/185508/136195549-4c128689-24a2-44c6-a4ad-accf963d367d.png)

Note thousands copies of error message.

**Comment by chetan-rathore** [comment]
Hi @hrw,

We have updated the test to handle the case when the extended capability is not present.
Fixed in this checkin: #189

Thanks,
ACS team
